sale_margin_delivered/views/sale_margin_delivered_view.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 Tecnativa - Sergio Teruel
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl-3). -->
+<odoo>
+
+<record id="view_order_form" model="ir.ui.view" >
+    <field name="model">sale.order</field>
+    <field name="inherit_id" ref="sale.view_order_form"/>
+    <field name="groups_id" eval="[(4,ref('sale_margin_security.group_sale_margin_security'))]"/>
+    <field name="arch" type="xml">
+       <xpath expr="//field[@name='order_line']/tree//field[@name='tax_id']" position="before">
+           <field name="margin" string="Margin"/>
```

Comment:
Isn't this field already present in standard `sale_margin`?

Reply:
Only globally, not per line